### PR TITLE
Update anime-list-master.xml

### PR DESCRIPTION
**Pull request by krbrs**
Fixed Mapping of Idolmaster Cinderella Girls Gekijou S01-S04 Specials and Idolmaster Cinderella Girls 2015 Specials

Unsure about the usage of manual mapping or offset mapping, so changed to manual mapping

### CODE REVIEW

File: anime-list-master.xml [hunk 1]
```diff
@@ -30552,6 +30552,9 @@
   </anime>
   <anime anidbid="11135" tvdbid="289763" defaulttvdbseason="1" episodeoffset="13" tmdbid="" imdbid="">
     <name>The Idolmaster: Cinderella Girls (2015)</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-2;2-3;3-4;</mapping>
```
Comment (reconman) [inline anime-list-master.xml:30556]:
```suggestion
      <mapping anidbseason="0" tvdbseason="0" offset="1"/>
```

File: anime-list-master.xml [hunk 2]
```diff
@@ -36387,8 +36390,7 @@
   <anime anidbid="13274" tvdbid="326337" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Idolmaster Cinderella Girls Gekijou (2017)</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;14-0;15-0;16-0;</mapping>
-      <mapping anidbseason="0" tvdbseason="0" start="1" end="13" offset="13"/>
+      <mapping anidbseason="0" tvdbseason="0">;1-14;2-15;3-16;4-17;5-18;6-19;7-20;8-21;9-22;10-23;11-24;12-25;13-26;</mapping>
```
Comment (reconman) [inline anime-list-master.xml:36393]:
```suggestion
      <mapping anidbseason="0" tvdbseason="0" start="1" end="13" offset="13">;14-44;15-45;16-46;</mapping>
```

File: anime-list-master.xml [hunk 3]
```diff
@@ -38070,7 +38072,7 @@
   <anime anidbid="13859" tvdbid="326337" defaulttvdbseason="3" episodeoffset="" tmdbid="" imdbid="">
     <name>Idolmaster Cinderella Girls Gekijou (2018)</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0" start="1" end="8" offset="26"/>
+      <mapping anidbseason="0" tvdbseason="0">;1-27;2-28;3-29;4-30;5-31;6-32;7-33;8-47;9-48;10-49;</mapping>
```
Comment (reconman) [inline anime-list-master.xml:38075]:
```suggestion
      <mapping anidbseason="0" tvdbseason="0" start="1" end="7" offset="26">;8-47;9-48;10-49;</mapping>
```

File: anime-list-master.xml [hunk 4]
```diff
@@ -39928,8 +39930,9 @@
   <anime anidbid="14515" tvdbid="hentai" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Shishunki Sex</name>
   </anime>
-  <anime anidbid="14516" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="14516" tvdbid="326337" defaulttvdbseason="4" episodeoffset="" tmdbid="" imdbid="">
     <name>Idolmaster Cinderella Girls Gekijou (2019)</name>
+	  <mapping anidbseason="0" tvdbseason="0">;1-34;2-35;3-36;4-37;5-38;6-39;7-40;8-50;9-51;10-52;</mapping>
```
Comment (reconman) [inline anime-list-master.xml:39935]:
```suggestion
	  <mapping anidbseason="0" tvdbseason="0" start="1" end="7" offset="33">;8-50;9-51;10-52;</mapping>
```

**Review by reconman — CHANGES_REQUESTED** [review]
I think my suggested format works fine. If you want to really make sure that it works, Hama (if you're using it) supports overriding the anime-list.xml entries.

**Comment by krbrs** [comment]
Seems like I still have problems using github correctly, sorry 😅 
Do I have to resolve/commit your suggestions or somebody else since I can't?

By override you mean using the anime-list-corrections.xml? I tried my filters there first. But somehow the special episodes still get listed as regular episodes - so for example Plex shows 2 Files for Episode 1 for Cinderella Girls Gekijou 2017 and so on. I already removed the whole show from the Library and re-added it with no change.

**Comment by reconman** [comment]
> Do I have to resolve/commit your suggestions or somebody else since I can't?

Github has troubles with the size of the XML file, so the buttons don't work. You have to copy my suggestions into the file.

> By override you mean using the anime-list-corrections.xml?

Yes, I do.

> But somehow the special episodes still get listed as regular episodes - so for example Plex shows 2 Files for Episode 1 for Cinderella Girls Gekijou 2017 and so on.

How does your folder and file structure look like? Can you post an image of your full directory tree?

Are you using `[anidb2-xxx]` folder suffixes? If you don't use anidb2, the anime-lists.xml will not be used.

**Comment by krbrs** [comment]
I changed the files and pushed a new commit with your suggested changes.

I tried creating a new library with only one series in it for testing. Regardless regular folder structure is:

driveletter:\anime_name-folder\files (e.g. R:\Idolmaster Cinderella Girls Gekijou (2017)\[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_01_(1920x1080_H264)_[57EC6284].mkv)

HAMA Logs scan the files but see them as regular episodes in their logs:

=============================================================================================================================================================
Library: 'Anime-Test', root: 'R:\', path: 'Idolmaster Cinderella Girls Gekijou (2017)', files: '26', dirs: '0'
Plex scan start: 2021-02-12 17:23:58,260000
=============================================================================================================================================================
-------------------------------------------------------------------------------------------------------------------------------------------------------------
misc_count: {'S10': 1, 'Cinderella': 26, 'Girls': 26, '02': 1, '03': 1, '01': 1, '06': 1, '07': 1, '04': 1, '05': 1, 'S13': 1, 'Gekijou': 26, '08': 1, '09': 1, 'S08': 1, 'S09': 1, 'Idolmaster': 26, '11': 1, '10': 1, '13': 1, '12': 1, 'S01': 1, 'S02': 1, 'S03': 1, 'S04': 1, 'S05': 1, 'S06': 1, 'S07': 1, 'S12': 1, 'S11': 1}
misc_words: ['Cinderella', 'Idolmaster', 'Gekijou', 'Girls']
-------------------------------------------------------------------------------------------------------------------------------------------------------------
"Idolmaster Cinderella Girls Gekijou (2017)" s01e001                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_01_(1920x1080_H264)_[57EC6284].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e002                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_02_(1920x1080_H264)_[29486F8F].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e003                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_03_(1920x1080_H264)_[FB83D768].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e004                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_04_(1920x1080_H264)_[CC693E63].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e005                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_05_(1920x1080_H264)_[4558E072].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e006                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_06_(1920x1080_H264)_[485C328E].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e007                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_07_(1920x1080_H264)_[F7053580].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e008                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_08_(1920x1080_H264)_[D64B9DC3].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e009                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_09_(1920x1080_H264)_[E1AB35A4].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e010                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_10_(1920x1080_H264)_[00F8D770].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e011                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_11_(1920x1080_H264)_[3296CDD1].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e012                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_12_(1920x1080_H264)_[75050762].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e013                         "SERIES_RX-2" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_13_(1920x1080_H264)_[F8E4C9D7].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e001                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S01_(1920x1080_H264)_[1F5E5753].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e002                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S02_(1920x1080_H264)_[BAF7B21D].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e003                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S03_(1920x1080_H264)_[1058DB38].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e004                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S04_(1920x1080_H264)_[90BF4A37].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e005                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S05_(1920x1080_H264)_[B4C913B1].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e006                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S06_(1920x1080_H264)_[043FE6E5].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e007                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S07_(1920x1080_H264)_[A264BDBA].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e008                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S08_(1920x1080_H264)_[236A88D4].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e009                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S09_(1920x1080_H264)_[29E9ADA5].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e010                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S10_(1920x1080_H264)_[D6F44DC7].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e011                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S11_(1920x1080_H264)_[81823E40].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e012                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S12_(1920x1080_H264)_[43B872C7].mkv"
"Idolmaster Cinderella Girls Gekijou (2017)" s01e013                         "ANIDB_RX-6" "" "[HorribleSubs]_Idolmaster_Cinderella_Girls_Gekijou_(2017)_-_S13_(1920x1080_H264)_[B301ED13].mkv"
=============================================================================================================================================================
Plex scan end: 2021-02-12 17:23:58,321000

I guess i need to open an issue on their repository for this if it is not an issue with the lists anymore.

I am not using the suffixes. Do you know if it is possible to add them automatically (i use Shoko for moving and organizing the files)?

**Comment by reconman** [comment]
I don't know enough about Absolute Series Scanner to be able to do find the source of your problem, but can you check if moving the specials to a `Specials` folder changes anything?

And be sure to create the issue in the ASS repository, not in the Hama repository.

**Review by reconman — APPROVED** [review]

**Pull request merged**